File: java/Solutions/fizzBuzzClass.java
package Solutions;

import java.util.ArrayList;
import java.util.List;

public class fizzBuzzClass {
    public fizzBuzzClass() {
        fizzBuzz(3);
    }
        public List<String> fizzBuzz(int n) {
        List<String> result = new ArrayList<String>();
        for(int i = 1;i < n+1; i++){
            String r = "";
            r += i % 3 == 0 ? "Fizz": "";
            r += i % 5 == 0 ? "Buzz": "";
            if (r.isEmpty())
                r = i+"";
            result.add(r);
        }
        return result;
    }
}

File: C#/Solutions/FizzBuzzClass.cs
class FizzBuzzClass
{
    string check (int num)
    {
        if (num % 3 == 0 && num % 5 == 0) return "FizzBuzz";
        if (num % 3 == 0) return "Fizz";
        if (num % 5 == 0) return "Buzz";
        return num.ToString();
    }
    
    public IList<string> FizzBuzz(int n)
    {
        return Enumerable.Range(1,n+1).Select(check).ToList();
    }
}